docs/operating/cdc.md

```diff
@@ -180,3 +180,27 @@ limitations for CDC processing:
 
 Transactions committed after version `0.16.29` are fully compatible with CDC and do not require
 the `history` flag.
+
+
+## Using CDC in Production
+
+### High Availability
+
+The CDC job is single instance. Starting a second `tigerbeetle cdc` publishing to the same queue 
```

Comment:
> publishing to the same queue

Let's try to clarify it: we lock by `cluster_id`, which means 1:1 TigerBeetle cluster to RMQ instance/vhost.